modules/ROOT/partials/integrations/react-quick-start.adoc

```diff
@@ -79,67 +167,440 @@ export default function App() {
+
+export default function BundledEditor(props) {
+  const {init, ...rest} = props;
+  // note that skin and content_css is disabled to avoid the normal
+  // loading process and is instead loaded as a string via content_style
+  return (
+    <Editor
+      init={{
+        ...init,
+        skin: false,
+        content_css: false,
+        content_style: [contentCss, contentUiCss, init.content_style || ''].join('\n'),
+      }}
+      {...rest}
+    />
+  );
+}
 ----
 
-endif::[]
-ifeval::["{productSource}" == "package-manager"]
-. Bundle {productname} with the React application using a module loader (such as Webpack).
+. Using a text editor, open `+./src/App.js+` and replace the contents with:
 +
---
-:depth: 1
-include::partial$integrations/bundling-integration.adoc[]
---
+[source,jsx]
+----
+import React, { useRef } from 'react';
+import BundledEditor from './BundledEditor'
 
+export default function App() {
+  const editorRef = useRef(null);
+  const log = () => {
+    if (editorRef.current) {
+      console.log(editorRef.current.getContent());
+    }
+  };
+  return (
+    <>
+      <BundledEditor
+        onInit={(evt, editor) => editorRef.current = editor}
+        initialValue='<p>This is the initial content of the editor.</p>'
+        init={{
+          height: 500,
+          menubar: false,
+          plugins: [
+            'advlist', 'anchor', 'autolink', 'help', 'image', 'link', 'lists',
+            'searchreplace', 'table', 'wordcount'
+          ],
+          toolbar: 'undo redo | blocks | ' +
+            'bold italic forecolor | alignleft aligncenter ' +
+            'alignright alignjustify | bullist numlist outdent indent | ' +
+            'removeformat | help',
+          content_style: 'body { font-family:Helvetica,Arial,sans-serif; font-size:14px }'
+        }}
+      />
+      <button onClick={log}>Log editor content</button>
+    </>
+  );
+}
+----
+endif::[]
 endif::[]
 ifeval::["{productSource}" == "zip"]
-. {productname} can be self-hosted by either: xref:deploying-tinymce-independent[Deploying {productname} independent of the React application], or xref:bundle[Bundling {productname} with the React application].
+ifeval::["{productUse}" == "host"]
+. Install the `+tinymce-react+` package and save it to your `+package.json+` with `+--save+`.
++
+[source,sh]
+----
+npm install --save @tinymce/tinymce-react
+----
 
-* [[deploying-tinymce-independent]] Deploying {productname} independent of the React application. To use a {productname} instance that has been deployed independent of the React application, use: xref:using-the-tinymcescriptsrc-prop[the `+tinymceScriptSrc+` prop], or xref:using-an-html-script-tag[an HTML `+<script>+` tag].
+. Unzip the content of the `+tinymce/js+` folder from the https://www.tiny.cloud/my-account/downloads/[{productname} zip] into the `+public+` folder. Afterwards the directory listing should be similar to below:
```

Comment:
```suggestion
. Unzip the content of the `+tinymce/js+` folder from the link:{download-enterprise}[{productname} zip] into the `+public+` folder. Afterwards the directory listing should be similar to below:
```
We should be using the global attributes for links like this as it makes it easier to change if the website structure changes (which has happened a few times).

Reply:
Also I've just noticed when looking up the above attribute, that we also have ZIP downloads for community via https://www.tiny.cloud/get-tiny/self-hosted/. It's doubtful anyone would use that when they can use NPM though, but that might be why this wasn't specific previously 🤷